xapian-applications/omega/templates/inc/anyallradio

```diff
@@ -1,2 +1,2 @@
-<input type="radio" name="DEFAULTOP" value="or $if{$eq{$defaultop,or}",CHECKED}> Matching any words
-<input type="radio" name="DEFAULTOP" value="and $if{$eq{$defaultop,and}",CHECKED}> Matching all words
+<input type="radio" name="DEFAULTOP" value="or" $if{$eq{$defaultop,or},checked}> Matching any words
```

Comment:
Ideally, moving the quote would have happened in either the quote attribute or fix errors commit.